server/src/main/java/io/littlehorse/common/model/getable/objectId/TenantIdModel.java

```diff
@@ -44,7 +44,7 @@ public String toString() {
 
     @Override
     public void initFromString(String storeKey) {
-        this.id = storeKey;
+        this.id = storeKey.split("/")[1];
```

Comment:
Why is this necessary? This looks like it could permit many bugs wherein I pass in a `WfSpecId` and get out a `TenantId` for example.

Reply:
When we are iterating the result of a range scan, we pass the full store key to the `ObjectIdModel#fromString` method. This is very common with the other ObjectId objects see `TaskRunIdModel.initFromString` , and I believe we should do the same for this `TenantIdModel`. 
Could you please explain to me how this can allow many bugs that are not present in the previous version of this class?